doc/nrf/app_dev/device_guides/nrf54l/fota_update.rst

```diff
@@ -0,0 +1,241 @@
+.. _ug_nrf54l_developing_ble_fota:
+
+FOTA updates on nRF54L Series devices
+#####################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. fota_upgrades_intro_start
+
+|fota_upgrades_def|
+You can also use FOTA updates to replace the application.
+See the :ref:`app_dfu` page for general Device Firmware Update (DFU) information, such as supported methods for sending and receiving updates on the device.
+
+.. note::
+   For more information about introducing immutable MCUboot bootloader, refer to :ref:`ug_bootloader_adding_sysbuild_immutable_mcuboot`.
+
+.. fota_upgrades_intro_end
+
+.. _ug_nrf54l_developing_ble_fota_steps:
+
+FOTA over Bluetooth Low Energy
+******************************
+
+.. fota_upgrades_over_ble_intro_start
+
+FOTA updates are supported using MCUmgr's Simple Management Protocol (SMP) over Bluetooth®.
+The application acts as a GATT server and allows the connected Bluetooth Central device to perform a firmware update.
+To use FOTA over Bluetooth LE, samples must support Bluetooth peripheral role (:kconfig:option:`CONFIG_BT_PERIPHERAL`).
+
+The application supports SMP handlers related to:
+
+* Image management.
+* Operating System (OS) management used to reboot the device after the firmware upload is complete.
+* Erasing settings partition used to ensure that a new application is not booted with incompatible content in the settings partition written by the previous application.
+
+To enable support for FOTA updates, do the following:
+
+* Enable the :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU` Kconfig option, which implies configuration of the following:
+
+   * All of the SMP command handlers mentioned in the previous paragraph.
+   * SMP BT reassembly feature.
+   * The :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP` Kconfig option automatically extends the Bluetooth buffers, which allows to speed up the FOTA transfer over Bluetooth, but also increases RAM usage.
+
+    .. note::
+       The :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU` Kconfig option enables the device to use MCUmgr for performing firmware over-the-air (FOTA) updates using Bluetooth LE.
+       It can be used along with other samples, and is meant as a demonstration of the default DFU configuration over Bluetooth.
+
```

Comment:
It is super important from application security perspective to add the one more note here after note on Line 46. The text for the note should be:

https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-18231/nrf/samples/bluetooth/mesh/dfu/distributor/README.html#:~:text=use%20it.-,Note,accessing%20SMP%20characteristics.%20See%20SMP%20over%20Bluetooth%20authentication%20for%20more%20information.,-Testing

> Note: To prevent an unauthenticated access to the device over SMP, it is strongly recommended to enable the [CONFIG_MCUMGR_TRANSPORT_BT_PERM_RW_AUTHEN](https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-18231/kconfig/index.html#CONFIG_MCUMGR_TRANSPORT_BT_PERM_RW_AUTHEN) option. This will enforce a remote device to initiate a pairing request before accessing SMP characteristics. See [SMP over Bluetooth authentication](https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-18231/nrf/samples/bluetooth/mesh/dfu/distributor/README.html#smp-over-bluetooth-authentication) for more information.